exercises/pythagorean-triplet/src/test/java/PythagoreanTripletTest.java

```diff
@@ -20,6 +20,135 @@ public void shouldCalculateSum() {
         assertEquals(expected, actual);
     }
 
+    @Test
```

Comment:
Hi @princerachit - thanks for getting this done.

As per our POLICIES.md, can we ignore all non-initial tests?

Thanks 🙂